packages/core-data/src/actions.js

```diff
@@ -448,9 +459,11 @@ export const redo =
  *
  * @return {Object} Action object.
  */
-export function __unstableCreateUndoLevel() {
-	return { type: 'CREATE_UNDO_LEVEL' };
-}
+export const __unstableCreateUndoLevel =
+	() =>
+	( { select } ) => {
+		select.getUndoManager().record();
```

Comment:
What does this do? I don't see this method in the API.

Reply:
I renamed to "addRecord" good catch :) Also, since it doesn't break anything I'm guessing we're not using this function maybe? I wonder if we can remove it at some point :P

Reply:
Looks like we are using it in two places, but maybe the codepaths are dead

Reply:
For some reason I thought this was the same as `__unstableMarkLastChangeAsPersistent`, but I guess not